main/acle.md

```diff
@@ -1503,21 +1508,27 @@ instructions include AES{E, D}, SHA1{C, P, M} and others. This also implies
 
 #### AES extension
 
-`__ARM_FEATURE_AES` is defined to 1 if the AES Crypto instructions from
-Armv8-A are supported and intrinsics targeting them are available. These
-instructions include AES{E, D}, AESMC, AESIMC and others.
+`__ARM_FEATURE_AES` is defined to 1 if the AES Crypto instructions
+from Armv8-A are supported and intrinsics targeting them are
+available. These instructions are identified by `FEAT_AES` and
+`FEAT_PMULL` in [[ARMARMv8]](#ARMARMv8), and they include AES{E, D},
+AESMC, AESIMC and others.
 
 #### SHA2 extension
 
-`__ARM_FEATURE_SHA2` is defined to 1 if the SHA1 & SHA2 Crypto instructions
-from Armv8-A are supported and intrinsics targeting them are available. These
-instructions include SHA1{C, P, M} and others.
+`__ARM_FEATURE_SHA2` is defined to 1 if the SHA1 & SHA2 Crypto
+instructions from Armv8-A are supported and intrinsics targeting them
+are available. These instructions are identified by `FEAT_SHA1` and
+`FEAT_SHA256` in [[ARMARMv8]](#ARMARMv8), and they include SHA1{C, P,
+M} and others.
```

Comment:
I guess this is a pre-existing problem with the text, but the example instructions here look wrong? FEAT_SHA256 includes instructions like SHA256H, SHA256H2 etc

Reply:
Ack - I have added these instructions, with `...` to avoid having to add all of them (see commit message)